torch/__future__.py

```diff
@@ -0,0 +1,15 @@
+"""
+This global flag controls whether to change the existing parameters
+in-place instead of assigning new tensors to the parameters when
+converting a module using `module._apply(fn)`.
+
+Default: False
+"""
+TORCH_FLAG_overwrite_module_params_on_conversion_ = False
```

Comment:
I'm not sure if I'm supposed to use the functions below or write this?  Or what `TORCH_FLAG` is supposed to mean.  Can you just start this with a "_".

Reply:
Fixed.